app/helpers/cart_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module CartHelper
+  def get_line_item_price(line_item)
```

Comment:
Nice helper, but we need to include a discount in the future.

Reply:
Exactly 👍